guides/google-tag-manager.md

```diff
@@ -0,0 +1,28 @@
+# Google Tag Manager
+> Google Tag Manager integration into a future ffc service requires contacting the **TBD** team. They will create the necessary Google Tag Manager containers needed to be setup for the new ffc service for Non-Production and Production. The developer will need to create a google account using their DEFRA email address. The developer can create a Google account without using Gmail here - https://accounts.google.com/SignUpWithoutGmail.
```

Comment:
It might be worth adding a note about when `TBD team` might be decided or who to contact in the interim.

Reply:
Agreed.